lib/index.js

```diff
@@ -173,7 +173,7 @@ function logEvent(event, context, err, result, options, moesifController) {
 
   logMessage(options.debug, 'logEvent', 'created request: \n' + JSON.stringify(logData.request));
   var safeRes = result || {};
-  logData.response.time = new Date(Math.max(logData.request.time.getTime(), Date.now()));
+  logData.response.time = Math.max(new Date(logData.request.time).getTime(), Date.now());
```

Comment:
you have tested, Math.max works for Dates? you have comparison fcn specified somewhere Or available in the library itself?

Reply:
Yes, I've tested. Both getTime() and Date.now() returns epoch time (number) and Math.max is available in library and works for number comparison.